debian/control.in

```diff
@@ -14,7 +14,7 @@ Homepage: http://www.openucx.org
 
 Package: @PACKAGE@
 Section: libs
-Depends: ${shlibs:Depends}, ${misc:Depends}
+#Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
why comment out and not remove?

Reply:
If for some reasons we'll decide to revert it back for some reasons in the future.

Reply:
we have git history for this.. we can add comment that the package does not have dependencies to allow running on different systems by relying on UCX dynamic modules

Reply:
Removed.